lib/validators/createWebLedgerRecord.js

```diff
@@ -90,8 +91,9 @@ module.exports = async ({
 async function _validateDid({
   basisBlockHeight, ledgerNode, validatorInput, validatorParameterSet
 }) {
-  const didDocument = new VeresOneDidDoc({doc: validatorInput.record});
-  let result = await didDocument.validateDid({mode: cfg.environment});
+  const didDocument = validatorInput.record;
+  let result =
+    await VeresOneDriver.validateDid({didDocument, mode: cfg.environment});
```

Comment:
```suggestion
  let result = await VeresOneDriver.validateDid(
    {didDocument, mode: cfg.environment});
```

Reply:
This seems like it would validate the whole DID doc (not just the DID, since the whole DID doc is passed) ... but can't really do anything about the naming here. Just noticing.